srcStatic/GameModules/ConsoleModule.cpp

```diff
@@ -8,14 +8,15 @@
 
 ConsoleModule::ConsoleModule(const std::string& moduleName) : DllModule(moduleName)
 {
-	moduleDirectory = (fs::path(GetExeDirectory()) / moduleName).string() + "\\";
+	moduleDirectory = moduleName + "\\";
```

Comment:
Maybe this class level variable should be renamed relativeModuleDirectory

Reply:
I'm fine either way. I went with the convention that `Directory()` returns a relative path, so things kind of match here.